docs/quickstarts/rosa-grant-ocm-role-access/metadata.yml

```diff
@@ -0,0 +1,5 @@
+kind: QuickStarts
+name: rosa-edit-cluster-name
```

Comment:
@aldiazRH I don't think this is the name we want - should be `rosa-grant-ocm-role-access`

Reply:
Thanks for the catch, I've updated it.